packages/ui/form/Select.tsx

```diff
@@ -66,7 +66,7 @@ export function UnstyledSelect<
   Option,
   IsMulti extends boolean = false,
   Group extends GroupBase<Option> = GroupBase<Option>
->({ className, ...props }: SelectProps<Option, IsMulti, Group>) {
+>({ ...props }: SelectProps<Option, IsMulti, Group>) {
```

Comment:
Let's revert this. I think we don't want to pass `className` with the rest of `...props`. In these cases removing `className` will affect `props` structure.